src/legacy/core_plugins/kibana/public/dashboard/panel/__tests__/panel_state.ts

```diff
@@ -17,41 +17,50 @@
  * under the License.
  */
 
+// @ts-ignore
 import expect from 'expect.js';
-
 import { createPanelState } from '../panel_state';
 
-function createPanelWithDimensions(x, y, w, h) {
+function createPanelWithDimensions(x: number, y: number, w: number, h: number) {
```

Comment:
Could add the return type of this function here as well.

Reply:
Awesome! Yes this is working in tests too when I run this locally which is great